.codeclimate.yml

```diff
@@ -0,0 +1,11 @@
+engines:
+  gofmt:
+    enabled: true
+  golint:
+    enabled: true
+  golint:
```

Comment:
Doh. This happens when you skip code reviews :)

Thanks for catching this!

I've created https://github.com/youtube/vitess/pull/2844 to fix it.